CDS/src/org/icpc/tools/cds/presentations/PresentationWebSocket.java

```diff
@@ -105,7 +104,7 @@ public void onOpen(Session session, EndpointConfig config) {
 			}
 		} else if (!"any".equals(role)) {
 			boolean isBlue = info.isBlue;
-			if (((Role.PRES_ADMIN.equals(role)) && !isAdmin) || (Role.BLUE.equals(role) && !(isAdmin || isBlue))) {
+			if ((("presAdmin".equals(role)) && !isAdmin) || ("staff".equals(role) && !(isAdmin || isBlue))) {
```

Comment:
What's the reason we are now hardcoding the role names? Shouldn't we still have a constant somewhere to make it easier to change it?

Reply:
Fixed in new commit. I was hesitating before b/c some of them aren't 'standard', but added these to IAccount since they are account types.